src/User32.Desktop/User32.cs

```diff
@@ -109,6 +128,12 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, StringBuilder lParam);
+
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, uint wParam, int lParam);
```

Comment:
That overload is actually incorrect. It could in theory unbalance the stack and crash the application.
It's not the case on Intel / AMD x64 because the ABI specifies that it's fastcall with the 4 first parameters in registers but the other 32 bits of the register passed as lParam to the message could contain anything, it won't crash but the emitted message will potentially be nonsensical.

I'm also not a fan about adding new overloads to `SendMessage`, I understand that having the marshaler do the string allocation (like in the previous `StringBuilder` case) is nice but we won't add all possible overloads present by hand, both wParam and lParam are intended as blackboxes with message-dependent content, and there are lot of messages.

That kind of feature seem more adapted as an helper. One possible solution would be :
* Having `SendMessage` expose both `wparam` and `lparam` as `void*` (what they really are) and still keep an `IntPtr` overload (`[Friendly]` should provide it automatically I think)
* Having a `SendMessage<TLParam, TWParam>(` helper that handle blittable structs, strings basic types and enums. object should also be supported but only if the passed in value is a null reference (To support the two overloads in the next point).
* Having both `SendMessage<TLParam>(` & `SendMessage<WParam>(` taking `object` for the non typed parameter. Theses overloads are here to support the case where one or the other is null.